### PR TITLE
트위터 연동시 스페이스 링크 체크

### PR DESCRIPTION
**Pull request by robin-maki**
(이벤트 코드 twitter_spacelink_2024)

### CODE REVIEW

**Comment by robin-maki** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1284** <a href="https://app.graphite.dev/github/pr/penxle/penxle/1284?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/1284?utm_source=stack-comment).

File: apps/penxle.com/prisma/schema.prisma [hunk 1]
```diff
@@ -583,6 +583,7 @@ model User {
   // PSL 선언을 위한 릴레이션들 (거의 쓰이지 않음)
   authoredTagWikiRevisions TagWikiRevision[]
   emailVerifications       UserEmailVerification[]
+  eventEnrollment          UserEventEnrollment[]
```
Comment (devunt) [inline apps/penxle.com/prisma/schema.prisma:586]: P1: 복수형이니 `eventEnrollments`

File: apps/penxle.com/prisma/schema.prisma [hunk 2]
```diff
@@ -620,6 +621,19 @@ model UserEmailVerification {
...
 }
 
+model UserEventEnrollment {
+  id         String    @id
+  userId     String    @map("user_id")
+  user       User      @relation(fields: [userId], references: [id])
+  eventCode  String    @map("event_code")
+  eligible   Boolean
+  createdAt  DateTime  @default(now()) @map("created_at") @db.Timestamptz
+  rewardedAt DateTime? @map("received_at") @db.Timestamptz
```
Comment (devunt) [inline apps/penxle.com/prisma/schema.prisma:631]: P1: 두개 이름이 달라요 (`rewardedAt` / `received_at`)

**Review by devunt — APPROVED** [review]

**Review by devunt — CHANGES_REQUESTED** [review]
엥 실수

**Review by devunt — APPROVED** [review]

**Pull request merged**